app/models/feedback.rb

```diff
@@ -22,6 +22,7 @@ class Feedback < ApplicationRecord
   after_create do
     ActionCable.server.broadcast "posts_#{self.post_id}", { feedback: FeedbacksController.render(locals: {feedback: self}, partial: 'feedback').html_safe }
     ActionCable.server.broadcast "api_feedback", { feedback: JSON.parse(FeedbacksController.render(locals: {feedback: self}, partial: 'feedback.json')) }
+    ActionCable.server.broadcast "topbar", { review: Post.feedbacks_count }
```

Comment:
Could probably reduce a little bit of load here by putting this in `Post.update_feedback_cache` and only firing it if https://github.com/Charcoal-SE/metasmoke/pull/127/files/4fda5d4fab5b76f0daf7c486b194a6d89cf0e6b3#diff-df2014f8b8976aed72db66eccb63e298R101 is true.